devices/YarpRobotLoggerDevice/src/YarpRobotLoggerDevice.cpp

```diff
@@ -32,6 +34,9 @@
 #include <yarp/eigen/Eigen.h>
 #include <yarp/os/BufferedPort.h>
 #include <yarp/profiler/NetworkProfiler.h>
+#include <yarp/os/all.h>
+
+#include <sstream>
```

Comment:
```suggestion
```



This is already included in line 16

Reply:
Fixed in [d92f7f4](https://github.com/ami-iit/bipedal-locomotion-framework/pull/796/commits/d92f7f4392f6d2c5b42a7494948cb755fe16bf80)